stm32cube/stm32h7xx/CMakeLists.txt

```diff
@@ -48,19 +51,24 @@ zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_LTDC_EX drivers/src/stm32h7xx_
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_MDIOS drivers/src/stm32h7xx_hal_mdios.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_MDMA drivers/src/stm32h7xx_hal_mdma.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_MMC drivers/src/stm32h7xx_hal_mmc.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_MMC_EX drivers/src/stm32h7xx_hal_mmc_ex.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_NAND drivers/src/stm32h7xx_hal_nand.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_NOR drivers/src/stm32h7xx_hal_nor.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_OPAMP drivers/src/stm32h7xx_hal_opamp.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_OPAMP_EX drivers/src/stm32h7xx_hal_opamp_ex.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_OSPI drivers/src/stm32h7xx_hal_ospi.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_OTFDEC drivers/src/stm32h7xx_hal_otfdec.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_PCD drivers/src/stm32h7xx_hal_pcd.c)
 zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_PCD_EX drivers/src/stm32h7xx_hal_pcd_ex.c)
+zephyr_library_sources_ifdef(CONFIG_USE_STM32_HAL_PSSI drivers/src/stm32h7xx_hal_pssi.c)
```

Comment:
Symbol USE_STM32_HAL_PSSI to be defined in Zephyr side

Reply:
this is added in modules/Kconfig.stm32 with https://github.com/zephyrproject-rtos/zephyr/pull/23259